Cargo.toml

```diff
@@ -109,3 +110,6 @@ overflow-checks = false
 all-features = true
 # opt-out of non-default targets
 targets = []
+
+[patch.crates-io]
+shared_str = { git = 'https://github.com/hdamron17/shared_str.git' }
```

Comment:
I think this can be removed now 0.1.1 is published :)

Reply:
I'm pretty sure I did this and you're looking at an older version...